examples/reprojection.js

```diff
@@ -4,6 +4,7 @@ import * as _ol_extent_ from '../src/ol/extent.js';
 import _ol_format_WMTSCapabilities_ from '../src/ol/format/WMTSCapabilities.js';
 import _ol_layer_Tile_ from '../src/ol/layer/Tile.js';
 import {get as getProjection} from '../src/ol/proj.js';
+import {register} from '../src/ol/proj/proj4.js';
```

Comment:
Same question about importing proj4.  Would be nice to have all our examples explicitly import dependencies eventually.